test/selenium/lib/view/sign_up/view.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+var SetUpAccessor = require('./accessors.js');
+
+var SuccessfulPageView = require('../successful_page/view.js');
+var successfulPageView;
+var isAlertPresent = false;
+
+function SetUpView(driver) {
+    this.driver = driver;
+    this.accessors = new SetUpAccessor(this.driver);
+    successfulPageView = new SuccessfulPageView(this.driver);
+};
+
+SetUpView.prototype = {
+
+    isSetUpView: function() {
+        return this.accessors.root;
+    },
+
+    confirmAllFieldsArePresent: function() {
+        this.passwordField();
```

Comment:
This line and the followings have no effect, and will always pass. The distinction between views and acessors is not clear. Acessors usually return WebElements, and view manipulate them.

I'd propose something like (the following comments are not needed in the final code):
```js
get areAllFieldsPresent() {
  var expectedElements = [this.accessors.passwordField, this.accessors.confirmPassword, this.accessors.submitButton] // We get WebElements so we can interact with.
  var promises = expectedElements.map((element) => { element.isDisplayed() }); // isDisplayed returns a promise (which is a bit a of a pain). We store every promise together, so we can wait on all of them
  return Promise.all((areDisplayed) => { // Here we wait on all of them to finish. Their results are store in the array "areDisplayed". Promise.all also returns a Promise
    return areDisplayed.reduce((previous, current) => { previous && current }) // Here we concatenate the array into one boolean 
  })
}
```
Side note: array functions `(foo) => { foo.bar }` are the equivalent of `function(foo) { return foo.bar }`. They only work on not too old browser (the one which support ES6) and NodeJS (> 4.x)